PlatShop.md

```diff
@@ -0,0 +1,7 @@
+Platinum (Plat) can be purchased with real life money in order to obtain global player buffs, quality of life tokens, and cosmentic items. 
+<br />
+Global player buffs will apply to the entire server for others to enjoy, as well as a special thank you in general chat when purchased. You can opt out of displaying your name by clicking "Anonymous Purchase". 20 Plat is equal to one minute of buff time. Buffs start out at level 2, but can upgrade to level 3 and 4 once certain time threshholds are met.
```

Comment:
Level 3 is 1 day. Level 4 is 1 week. Once the threshold is reached, the buff stays at that level including later extensions by different players or yourself